pandas/io/pytables.py

```diff
@@ -2499,9 +2507,16 @@ class Fixed:
     pandas_kind: str
     obj_type: Type[Union[DataFrame, Series]]
     ndim: int
+    parent: HDFStore
+    group: "Node"
     is_table = False
 
-    def __init__(self, parent, group, encoding=None, errors="strict", **kwargs):
+    def __init__(
+        self, parent: HDFStore, group: "Node", encoding=None, errors="strict", **kwargs
+    ):
+        assert isinstance(parent, HDFStore), type(parent)
+        assert _table_mod is not None  # needed for mypy
```

Comment:
Does this not have any impact on runtime? The global state management here is a little nuanced so just want to double check that it is valid to add to the constructor here, as on first glance it may not be set until methods are called by design

Reply:
It does not, we wouldn't get here if _table_mod were not initialized